net/portmapper/portmapper.go

```diff
@@ -876,6 +883,11 @@ var uPnPPacket = []byte("M-SEARCH * HTTP/1.1\r\n" +
 	"ST: ssdp:all\r\n" +
 	"MAN: \"ssdp:discover\"\r\n" +
 	"MX: 2\r\n\r\n")
+var uPnPIGDPacket = []byte("M-SEARCH * HTTP/1.1\r\n" +
```

Comment:
comment on this one, with reference to bug? (copy some of the comment from above)

Reply:
Added comment with reference to bug.